### PR TITLE
RDS-170: Gretel DGAN API Notebook

### PR DESCRIPTION
**Pull request by santhosh97**

### CODE REVIEW

File: docs/notebooks/create_synthetic_data_from_gretel_dgan_api.ipynb [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+   "source": [
+    "#Generate autocorrelation features from synthetic and real data and plot!\n",
+    "synthetic_acf = get_autocorr(generate_numpy_for_autocorr(df, seq_len))\n",
+    "acf = get_autocorr(generate_numpy_for_autocorr(synthetic_df, seq_len))\n",
+    "# Figure 1, autocorrelation\n",
+    "plt.plot(acf, label=\"real\")\n",
+    "plt.plot(synthetic_acf, label=\"generated\")\n",
+    "plt.xlabel(\"Time lag (days)\")\n",
+    "plt.ylabel(\"Autocorrelation\")\n",
+    "plt.title(\"Autocorrelation of Heartbeat 1 and Heartbeat 2\")\n",
```
Comment (belson17) [inline docs/notebooks/create_synthetic_data_from_gretel_dgan_api.ipynb:294]: Earlier it says the dataset is "oil", but here it says heartbeat 1 and 2, is that right?

File: docs/notebooks/create_synthetic_data_from_gretel_dgan_api.ipynb [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0a3d8f5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Now, let's visualize the probability distribution of each feature and it's respective synthetic data alternate.\n",
+    "for val in list(df.iloc[:,1:].columns):\n",
```
Comment (belson17) [inline docs/notebooks/create_synthetic_data_from_gretel_dgan_api.ipynb:213]: Minor - This gives a warning,
/usr/local/lib/python3.7/dist-packages/numpy/core/fromnumeric.py:3208: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  return asarray(a).size
/usr/local/lib/python3.7/dist-packages/matplotlib/cbook/__init__.py:1376: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  X = np.atleast_1d(X.T if isinstance(X, np.ndarray) else np.asarray(X))

File: docs/notebooks/create_synthetic_data_from_gretel_dgan_api.ipynb (hunk 1 above)
Comment (belson17) [inline docs/notebooks/create_synthetic_data_from_gretel_dgan_api.ipynb:294]: The autocorrelation of the real vs generated data look very different. 

Screen Shot 2022-08-15 at 2 12 31 PM

**Comment by zredlined** [comment]
@santhosh97 can you rename the notebook to something simpler? E.g. timeseries-DGAN.ipynb, or DoppelGANger-timeseries.ipynb?

**Pull request closed without merging**